### ISSUE TITLE
Add other users as co-writers to his blog. (For community based blogs) 

### ISSUE THREAD

**Issue by marc-mader00** [Plume-org/Plume#848]
I would like to use this software to build a community based blog. Could you somehow implement this feature? It would be really very cool if they would add this feature as soon as possible in one of the upcoming versions.

Thanks

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by KitaitiMakoto** [comment]
Yep, it’s nice and on our (long) milestone: https://github.com/Plume-org/Plume/issues/774
